src/Controller/Component/CacheComponent.php

```diff
@@ -54,13 +54,14 @@ public function shutdown(Event $event) {
 	 * @return bool|int|string
 	 */
 	protected function _isActionCachable() {
+		if (!$this->getController()->getRequest()->is('get')) {
```

Comment:
Why the order change?

Reply:
If no actions are defined POST requests would be marked as cachable. Not served by the middleware but still written to storage.

Reply:
Sounds like we need to port this also to master then before or after this merge.